lib/plugins/inspec-init/templates/profiles/aws/inspec.yml

```diff
@@ -6,11 +6,15 @@ copyright_email: you@example.com
 license: Apache-2.0
 summary: An InSpec Compliance Profile For AWS
 version: 0.1.0
-inspec_version: '>= 2.3.5'
+inspec_version: '~> 4'
 attributes:
 - name: aws_vpc_id
-  required: true
-  description: 'The Custom AWS VPC Id'
+  required: false
+  default: ''
```

Comment:
If you mean "there should be no value unless the user provides one", you should either remove this or comment it out. As-is, this will set an empty-string vpc-id, which is probably not what you want.

Reply:
This was so users didn't have to specify a VPC identifier e.g. 
```
   only_if { aws_vpc_id != ''}                                               # Only run this control if the `aws_vpc_id` atrtibute is provided.
```
There's a note and link to related documentation in the README also.

Reply:
Can we put a comment with a link to that documentation above this. My worry here is with people copypasta'ing the example for eternity.

Reply:
Thanks @miah I've added a comment to the `inspec.yml`.